src/simplicial-hott/08-covariant.rzk.md

```diff
@@ -1565,3 +1565,28 @@ commuting with the contravariant lifts.
     ( contravariant-fiberwise-transformation-application
         A x y f C D is-contravariant-C ϕ v)
 ```
+
+## Two sided discrete fibrations
+
+```rzk title="RS17, Definition 8.28"
+#def is-two-sided-discrete
+  (A B : U)
+  (C : A -> B -> U)
+  : U
+  :=
+    product
+      ((a : A) -> is-covariant B (\b -> C a b))
+      ((b : B) -> is-contravariant A (\ a -> C a b))
+```
+
+```rzk title="RS17, Proposition 8.29"
+#def is-two-sided-discrete-representable-is-segal
+  (A : U)
+  (is-segal-A : is-segal A)
```

Comment:
```suggestion
  ( A : U)
  ( is-segal-A : is-segal A)
```

Reply:
The convention is to place spaces after opening parentheses on new lines. This is to neatly align indented code, and highlight the tree structure of definitions in `rzk`.